src/htmx.js

```diff
@@ -127,10 +127,29 @@ return (function () {
             return "[hx-" + verb + "], [data-hx-" + verb + "]"
         }).join(", ");
 
+        /**
+         * @type {Array<RegExp>}
```

Comment:
Sure! I did that at first actually, you're right. Just changed now, but I kept `var` instead of `const` to be consistent with the actual code which supports IE 11.